README.md

```diff
@@ -1,3 +1,17 @@
+<p align="center">
```

Comment:
How come the html for Markdown?

Reply:
IMO, it looks good from its [preview](https://github.com/FeynmanZhou/fluentbit-operator#readme). I also see other projects use HTML in README to make the layout looks neater,